.gitignore

```diff
@@ -122,6 +122,9 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
 
+# VSCode configs
+/.vscode
```

Comment:
This seems a bit unrelated to this change. Are there some vscode configs we want to include in this repository ?

Reply:
it might make sense to enforce prettier as formatter